templates/search/list.html

```diff
@@ -16,61 +17,37 @@
   </form>
 </header>
 
-{% if users %}
-<div class=results>
-  <section>
+<section>
+  {% if users %}
+  <article>
     <h2>neomad finding</h2>
     <ul>
       {% for user in users %}
-        <li class=profile-intro>
-          <a href={{ user.slug }}>
-            <h2 name=username>{{ user.username }}</h2>
-            <figure>
-              <img class=avatar-img src={{ user.avatar }} width=100 height=100 onerror="this.src='{{ url_for('static', filename='images/avatar.png') }}'">
-            </figure>
-          </a>
-        </li>
+      <li class=profiles>
+        <a href='@{{ user.slug }}'>
+          <h2 name=username>{{ user.username }}</h2>
+          <figure>
+            <img class=avatar-img src={{ user.avatar }} width=100 height=100 onerror="this.src='{{ url_for('static', filename='images/avatar.png') }}'">
+          </figure>
+        </a>
```

Comment:
I agree with you @vinyll but if we create a component or macro for `avatar` we should do that in an other PR